### PR TITLE
chore: set bridge selected tokens and amount

### PR DESCRIPTION
**Pull request by micaelae**
## **Description**
Changes include
* redux actions to set the selected src/dest tokens and token amount
* redux selectors to get the selected src/dest tokens and token amount

[Open in GitHub Codespaces](https://codespaces.new/MetaMask/metamask-extension/pull/26212?quickstart=1)

## **Related issues**

Fixes:  https://consensyssoftware.atlassian.net/browse/METABRIDGE-866

## **Manual testing steps**

N/A. This doesn't change any user functionality, just setting up getters/setters

## **Screenshots/Recordings**

N/A, redux state changes will take effect after UI for inputs is implemented

## **Pre-merge author checklist**

- [X] I've followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Extension Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [X] I've completed the PR template to the best of my ability
- [X] I’ve included tests if applicable
- [X] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [X] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Review by BZahory — DISMISSED** [review]

**Review by ejwessel — DISMISSED** [review]

**Review by BZahory — APPROVED** [review]

File: ui/ducks/bridge/bridge.ts [hunk 1]
```diff
@@ -1,15 +1,22 @@
...
+  fromToken: SwapsTokenObject | SwapsEthToken | undefined;
+  toToken: SwapsTokenObject | SwapsEthToken | undefined;
+  fromTokenInputValue: string | undefined;
 };
 
 const initialState: BridgeState = {
   toChain: null,
+  fromToken: undefined,
+  toToken: undefined,
+  fromTokenInputValue: undefined,
```
Comment (infiniteflower) [inline ui/ducks/bridge/bridge.ts:19]: Do we want to align on a common initial value between `null` and `undefined`?

`toChain` is `null` but the rest are `undefined`.

Comment (infiniteflower) [inline ui/ducks/bridge/bridge.ts:19]: Do we want to align on a common initial value between `null` and `undefined`?

`toChain` is `null` but the rest are `undefined`.

**Review by infiniteflower — COMMENTED** [review]
Left a small comment

File: ui/ducks/bridge/bridge.ts (hunk 1 above)
Comment (micaelae) [reply ui/ducks/bridge/bridge.ts]: Good call - updated the default values to `null`

**Review by infiniteflower — APPROVED** [review]

**Review by ejwessel — APPROVED** [review]

**Pull request merged**